cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -394,7 +394,7 @@ The minimum index value is 0. The maximum index value is 2147483647.`)
 				"Where to redirect std error stream of a task. If not passed it proceeds to stdout, and is available via kubectl logs.")
 			o.SlurmFlagSet.StringVar(&o.Input, inputFlagName, "",
 				"What to pipe into the script.")
-			o.SlurmFlagSet.StringVar(&o.JobName, jobNameFlagName, "",
+			o.SlurmFlagSet.StringVarP(&o.JobName, jobNameFlagName, "J", "",
```

Comment:
I confirmed that the sbatch command has the same alias as well:

> -J, --job-name=<jobname>
Specify a name for the job allocation. The specified name will appear along with the job id number when querying running jobs on the system. The default is the name of the batch script, or just "sbatch" if the script is read on sbatch's standard input.